homeassistant/components/weatherkit/sensor.py

```diff
@@ -0,0 +1,79 @@
+"""WeatherKit sensors."""
+
+from datetime import date, datetime
+from decimal import Decimal
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfVolumetricFlux
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import API_KEY_CURRENT_WEATHER, DOMAIN
+from .coordinator import WeatherKitDataUpdateCoordinator
+from .entity import WeatherKitEntity
+
+SENSORS = (
+    SensorEntityDescription(
+        key="precipitationIntensity",
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfVolumetricFlux.MILLIMETERS_PER_HOUR,
+    ),
+    SensorEntityDescription(
+        key="pressureTrend",
+        device_class=SensorDeviceClass.ENUM,
+        icon="mdi:gauge",
+        options=["rising", "falling", "steady"],
+        translation_key="pressure_trend",
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Add sensor entities from a config_entry."""
+    coordinator: WeatherKitDataUpdateCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ]
+
+    sensors = [WeatherKitSensor(coordinator, description) for description in SENSORS]
+
+    async_add_entities(sensors)
```

Comment:
I think you can do something like this:
```suggestion
    async_add_entities(WeatherKitSensor(coordinator, description) for description in SENSORS)
```

Reply:
That's true, I just wanted to avoid making the lines too long. But if this doesn't make the linter upset then it's fine 👍